
This fixes the powerpc run-time arch detection ...

  - Rene Rebe <rene@exactcode.de>

diff -ur MPlayer-1.0pre5-orig/libavcodec/libpostproc/postprocess.c MPlayer-1.0pre5/libavcodec/libpostproc/postprocess.c
--- MPlayer-1.0pre5-orig/libavcodec/libpostproc/postprocess.c	2004-07-15 02:18:47.000000000 +0200
+++ MPlayer-1.0pre5/libavcodec/libpostproc/postprocess.c	2004-10-19 01:58:06.244243680 +0200
@@ -696,7 +696,7 @@
 #else
 #ifdef ARCH_POWERPC
 #ifdef HAVE_ALTIVEC
-        else if(c->cpuCaps & PP_CPU_CAPS_ALTIVEC)
+        if(c->cpuCaps & PP_CPU_CAPS_ALTIVEC)
 		postProcess_altivec(src, srcStride, dst, dstStride, width, height, QPs, QPStride, isColor, c);
         else
 #endif
